index.js

```diff
@@ -1,11 +1,35 @@
-function scuberGreetingForFeet(){
+function scuberGreetingForFeet(distance){
   // Write your code here!
+  //This function will return a price for a trip depending on the entered distance 
+
+  if (distance <= 400){
+    return "This one is on me!";
+  } else if (distance > 400 && distance <= 2000){
```

Comment:
You can exclude `distance > 400 &&` since it's already covered by the first part of the conditional.
```suggestion
  } else if (distance <= 2000){
```